operations/research-and-development/product/growth/ab-testing.md

```diff
@@ -0,0 +1,119 @@
+# Overview
+
+This document will provide an overview of Mattermost’s A/B Testing methodology. It consists of a sequence of steps and considerations to design valid A/B tests.
+
+## A/B Testing Methodology
+
+### Success Criteria
+First you must decide what the success criteria of your A/B test will be. The following questions must be answered in order to formulate a proper A/B testing hypothesis:
+
+1. What is the metric or measure you are trying to influence? (i.e. What action or outcome are you trying to influence?)
+  * This must be a concise definition that prevents the metric from changing over time.
+  * A/B tests tend to work best when working with proportions i.e. the percentage of users to take an action similar to conversion rates.
+2. Which direction are you trying to influence the measure (increase, decrease or no directional preference)?
+3. Can the measure be compared to historical benchmarks?
+  * Is it a static measure (one that will not change as more time passes)?
+  * Historical benchmarks simplify sample size requirement calculations and provide us with a better understanding of the time required to run a test (i.e. the amount of time required to collect enough data/reach a large enough sample).
+4. What is the minimum required relative change you need to see between the control and tests to consider implementing the changes?
+  * Example: A 20% minimum required relative change in users inviting members in the first 14 days where the latest (estimated control group’s) conversion rate is ~22%.
```

Comment:
Is 20% relative change something we'd recommend by default?

Reply:
I think it will vary a lot, but for conversion rates and proportion related success criteria metrics it should be a decent rule of thumb.

Reply:
+1, thanks!

Reply:
This has also been added in most recent commit. @jasonblais